src/Maker/MakeAuthenticator.php

```diff
@@ -403,11 +403,11 @@ public function configureDependencies(DependencyBuilder $dependencies, InputInte
     private function providerKeyTypeHint(): string
     {
         $reflectionMethod = new \ReflectionMethod(AbstractFormLoginAuthenticator::class, 'onAuthenticationSuccess');
-        $typeHint = (string) $reflectionMethod->getParameters()[2]->getType();
-        if ($typeHint) {
-            $typeHint .= ' ';
+
+        if (null !== $type = $reflectionMethod->getParameters()[2]) {
```

Comment:
This looks backwards to me? Shouldn't it be `null === $type`, then `return ''`, else (at the bottom) return `sprintf('%s ', $type->getName())`?

Reply:
good catch! refactored to be a bit more clear && fix my mistake.